ic-agent/src/agent/http_transport/reqwest_transport.rs

```diff
@@ -88,6 +89,15 @@ impl ReqwestTransport {
         }
     }
 
+    /// Use call v3 endpoint for update calls.
```

Comment:
Please explain a little more what the caller is signing up for. (And same for hyper)